packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/resources/dataservice/ModelConfigurationResource.java

```diff
@@ -0,0 +1,63 @@
+package software.uncharted.terarium.hmiserver.resources.dataservice;
+
+import org.eclipse.microprofile.openapi.annotations.tags.Tag;
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+import software.uncharted.terarium.hmiserver.models.dataservice.ModelConfiguration;
+import software.uncharted.terarium.hmiserver.proxies.dataservice.ModelConfigurationProxy;
+import software.uncharted.terarium.hmiserver.models.dataservice.Model;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import software.uncharted.terarium.hmiserver.utils.Converter;
+
+import javax.inject.Inject;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.util.*;
+
+@Path("/api/model_configurations")
+@Produces(MediaType.APPLICATION_JSON)
+@Tag(name = "Model Configuration REST Endpoints")
+public class ModelConfigurationResource {
+	@Inject
+	@RestClient
+	ModelConfigurationProxy proxy;
+
+	@GET
+	public Response getModelConfigurations() {
+		return proxy.getModelConfigurations();
+	}
+
+	@POST
+	@Consumes(MediaType.APPLICATION_JSON)
+	public Response createModelConfiguration(Object config) {
```

Comment:
Do we have a `config` POJO we can use?

Reply:
Actually I see below `ModelConfiguration` if thats the same can we use that here and anywhere else there is an `Object` reference?

Reply:
When I create - the config has to lack some attributes of the ModelConfig object such as id etc. So I just made it an object.

Reply:
I think you can use `@TSOptional` on those fields to have it not complain about them being issing

Reply:
True but what I am passing is more like a payload to create a model config, everywhere else those attributes shouldn't be optional. Do you think I should rename just the `config` variable to `createConfigPayload` or should I create another POJO along with that?

Reply:
Note ModelConfiguration actually doesn't quite work because down-stream systems cannot consume AMR format just yet. So we need to do a bit of hacky things and transforms  on the fly .... so in light of that not too horribly worried about Object-typings and related matters, but we should revisit next week.